tests/sparseml/pytorch/models/classification/test_resnet.py

```diff
@@ -63,24 +63,22 @@
         ("resnet50", False, True, resnet50),
         ("resnet50", True, False, resnet50),
         ("resnet50", "base", False, resnet50),
-        ("resnet50", "pruned-conservative", False, resnet50),
-        ("resnet50", "pruned-moderate", False, resnet50),
```

Comment:
both deprecated. stub cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned-moderate, imagenet is not there